Sources/PrimerSDK/Classes/User Interface/OAuth/OAuthViewController.swift

```diff
@@ -23,42 +23,39 @@ internal class OAuthViewController: PrimerViewController {
     deinit {
         log(logLevel: .verbose, message: "🧨 destroyed: \(self.self)")
     }
```

Comment:
I will revert this file to its original state. Believe it makes more sense to maintain smaller separate view controllers with better abstraction. Wdyt?

Reply:
In the future I'd like to completely remove this controller, which is actually presented between the previous view controller and the webview controller.

Let it be like that for now.